content/cpp/concepts/pointers/pointers.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: "Pointers"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags: 
+  - "Pointers"
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A pointer is a speacial type variable that stores address of another variable in it. 
+
+## Declaration and Initialization of Pointer variable
+
+A pointer variable points to the same data type and is created with the `*` operator. Then the address of another variable is stored in it.
+
+```cpp
+string game = "Cricket";  // A game variable of type string
+string* ptr = &game;    // A pointer variable, with the name ptr, that stores the address of game
```

Comment:
I would either shorten the comment or move it directly below line 21, on its own line.